jp/src/edu/hillel/lesson23/Builder/CarBuilder.java

```diff
@@ -0,0 +1,26 @@
+package edu.hillel.lesson23.Builder;
+
+public class CarBuilder {
```

Comment:
Usually, builder is creating inside a class as inner class.
There is a not mistake, just a best practices

Reply:
Changed